src/main/java/lotto/view/OutputView.java

```diff
@@ -0,0 +1,72 @@
+package lotto.view;
+
+import lotto.domain.Lotto;
+import lotto.domain.PrizeGrade;
+
+public class OutputView {
+
+    public static final String ERROR_MESSAGE = "[ERROR] ";
+    private static final String REQUEST_LOTTO_PRICE = "구입금액을 입력해 주세요.";
+    private static final String SYSTEM_LOTTO_COUNT = "개를 구매했습니다.";
+    private static final String REQUEST_WINNING_NUMBER_BEFORE = "\n지난 주 당첨 번호를 입력해주세요.(ex 1, 2, 3, 4, 5, 6)";
+    private static final String REQUEST_BONUS_BALL_NUMBER = "보너스 볼을 입력해주세요.";
+    private static final String WINNING_STATS = "\n당첨 통계\n------------";
+
+    public static void printExceptionMessage(String exceptionMessage) {
+        System.out.println(ERROR_MESSAGE + exceptionMessage);
+    }
+
+    public static void printRequestLottoPrice() {
+        System.out.println(REQUEST_LOTTO_PRICE);
+    }
+
+    public static void printLottoCount(int count) {
+        System.out.println(count + SYSTEM_LOTTO_COUNT);
+    }
+
+    public static void printWinningNumberBefore() {
+        System.out.println(REQUEST_WINNING_NUMBER_BEFORE);
+    }
+
+    public static void printBonusBallNumber() {
+        System.out.println(REQUEST_BONUS_BALL_NUMBER);
+    }
+
+    public static void printLottoStatisticTitle() {
+        System.out.println(WINNING_STATS);
+    }
+
+    public static void printLottoStatistic(PrizeGrade prizeGrade, int prizeGradeCount) {
+        if (prizeGrade == PrizeGrade.NONE) {
+            return ;
+        }
+        if (prizeGrade == PrizeGrade.SECOND) {
+            System.out.println(
+                (prizeGrade.getMatchCount() + 1) + "개 일치, 보너스 볼 일치(" + prizeGrade.getPrizeMoney()
+                    + "원) - " + prizeGradeCount + " 개");
```

Comment:
```suggestion
String PRINT_PRIZE_GRADE_RESULT_FORMAT = "%d개 일치, 보너스 볼 일치(%d원) - %d개";
        if (prizeGrade == PrizeGrade.SECOND) {
            System.out.println(String.format(PRINT_PRIZE_GRADE_RESULT_FORMAT
              , prizeGrade.getMatchCount() + 1
              , prizeGrade.getPrizeMoney()
              , prizeGradeCount));
```
`+`연산자로 문자열을 구성하는 것보다 `String.format`을 이용하는 게 더 깔끔하게 표현할 수 있습니다.

Reply:
- https://github.com/hochan222/java-lotto/commit/3b283e65cde4527d74e3f245b0681c70ae5bc38e [+ 연산자로 문자열을 구성하는 것보다는 String.format을 이용해보자.](https://github.com/hochan222/java-lotto/commit/3b283e65cde4527d74e3f245b0681c70ae5bc38e)
- https://github.com/hochan222/java-lotto/commit/7d8a65712104b7a4dfb17652b80a6b5ed368febc [System.out.println(String.format)을 System.out.printf으로 바꿔보자.](https://github.com/hochan222/java-lotto/commit/250a7d9ce25c6b76227ab3c3f58c489c2eb5ca91)

intellij는 printf를 추천해주네요!